librz/include/rz_bin.h

```diff
@@ -480,6 +482,8 @@ typedef struct rz_bin_section_t {
 	ut32 perm;
 	// per section platform info
 	const char *arch;
+	ut64 type;
+	int flags;
```

Comment:
I see flags was changed back to an `int` from a `ut64` in [7e13f8e](https://github.com/PinkNoize/rizin/commit/7e13f8eafc4d83b498f2589e4eb91f8215bd467b)
C only guarantees that an int is at least 16 bits

Was this intentional and if so what is the reason for it?